projects/packages/my-jetpack/_inc/components/my-jetpack-screen/index.jsx

```diff
@@ -11,16 +11,26 @@ import {
 	Col,
 } from '@automattic/jetpack-components';
 
+/**
+ * Internal dependencies
+ */
 import './style.scss';
 import ConnectionsSection from '../connections-section';
 import PlansSection from '../plans-section';
+import useAnalytics from '../../hooks/use-analytics';
 
 /**
  * The My Jetpack App Main Screen.
  *
  * @returns {object} The MyJetpackScreen component.
  */
 export default function MyJetpackScreen() {
+	const {
+		tracks: { recordEvent },
+	} = useAnalytics();
+	useEffect( () => {
+		recordEvent( 'jetpack_myjetpack_page_view' );
+	} );
```

Comment:
Although the component renders once so far, we should add an empty array as `useEffect` dependency to guarantee it will run just once the component mounts.

```suggestion
	} , [] );
```